src/redux/services/fileEntry.ts

```diff
@@ -430,7 +429,7 @@ export function reloadFile(
   if (isHelmChartFile(absolutePath)) {
     reloadHelmChartFile(fileEntry, state.fileMap, state.helmChartMap);
   } else if (shouldReloadResourcesFromFile(fileEntry, projectConfig)) {
-    reloadResourcesFromFileEntry(fileEntry, state, schemaVersion, userDataDir);
+    reloadResourcesFromFileEntry(fileEntry, state, projectConfig.k8sVersion || PREDEFINED_K8S_VERSION, userDataDir);
```

Comment:
I think it would be a good idea to create a function which returns the value of the k8sVersion. I see the same code bellow as well `projectConfig.k8sVersion || PREDEFINED_K8S_VERSION`

Reply:
you're right - will fix!